src/components/Progres/Progress.theme.ts

```diff
@@ -1,12 +1,16 @@
+import { mode } from '@chakra-ui/theme-tools'
+
 export const ProgressStyle = {
   parts: ['track', 'filledTrack', 'label'],
   // Styles for the base style
-  baseStyle: {
-    track: {},
+  baseStyle: (props: Record<string, any>) => ({
+    track: {
+      bg: mode('blackAlpha.100', 'whiteAlpha.100')(props),
+    },
     filledTrack: {
       borderRadius: 'full',
     },
-  },
+  }),
```

Comment:
?

Reply:
?

Reply:
Just house keeping, we never accounted for dark/light mode on the progress bar.